README.md

```diff
@@ -187,75 +143,76 @@ At the root of this repository, execute `setup.sh`. It will:
 
 ## Validation
 
-Validating these three deployments is done by simply executing `validate.sh`
-from the root of the directory.
-A successful output will contain:
+Validating these three deployments is done by executing:
+
+```console
+make validate
+```
+
+A successful output will look like this:
 ```console
 Validating Debian VM Webapp...
-Testing endpoint http://[ip-1]:8080
-Endpoint http://[ip-1]:8080 is responding.
+Testing endpoint http://35.227.149.80:8080
+Endpoint http://35.227.149.80:8080 is responding.
+**** http://35.227.149.80:8080/prime/10
 The sum of all primes less than 10 is 17
 The factorial of 10 is 3628800
 
 Validating Container OS Webapp...
-Testing endpoint http://[ip-2]:8080
-Endpoint http://[ip-2]:8080 is responding.
+Testing endpoint http://35.230.123.231:8080
+Endpoint http://35.230.123.231:8080 is responding.
+**** http://35.230.123.231:8080/prime/10
 The sum of all primes less than 10 is 17
 The factorial of 10 is 3628800
 
 Validating Kubernetes Webapp...
-Testing endpoint http://[ip-3]
-Endpoint http://[ip-3] is responding.
+Testing endpoint http://35.190.89.136
+Endpoint http://35.190.89.136 is responding.
+**** http://35.190.89.136/prime/10
 The sum of all primes less than 10 is 17
 The factorial of 10 is 3628800
 ```
 
+Of course, the IP addresses will likely differ for your deployment.
+
 ## Load Testing
 
-In a new console window, execute (using ip-3 from above)
+In a new console window, execute the following, replacing `[IP_ADDRESS]` with the IP address and port from your validation output from the previous step. Note that the Kubernetes deployment runs on port `80`, while the other two deployments run on port `8080`:
 ```console
-ab -c 120 -t 60  http://[ip-3]/prime/10000
+ab -c 120 -t 60  http://[IP_ADDRESS]/prime/10000
```

Comment:
those changes to parameterize the IP will make this better.

Reply:
Not sure I follow?

Reply:
Sorry, I think I was talking to myself on this one. But... let's use a legal bash variable `$IP_ADDRESS` instead of `[IP_ADDRESS]`.

Reply:
Per above, changed to `<>`